rpc/src/v1/impls/eth.rs

```diff
@@ -284,6 +284,11 @@ impl<C, SN: ?Sized, S: ?Sized, M, EM> Eth for EthClient<C, SN, S, M, EM> where
 		Ok(format!("{}", version))
 	}
 
+	fn chain_id(&self) -> Result<String, Error> {
```

Comment:
Chain id in the transactions is serialized as `Option<u64>`,  would be best to keep it constistent.
IMHO using a generic `String` response is a bit misleading. If we want to return a hex-encoded value then `Option<U64>` would be a right choice.

Note that using `Option` indicates that in case the chain id is missing `null` will be returned.